substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationParser.java

```diff
@@ -86,18 +86,22 @@ protected void checkAttributes(Map<String, Object> map, String type, Collection<
 
         if (unknownAttributes.size() > 0) {
             String message = "Unknown attribute(s) [" + String.join(", ", unknownAttributes) + "] in " + type;
-            if (strictConfiguration) {
-                throw new JSONParserException(message);
-            } else {
-                // Checkstyle: stop
-                System.err.println("WARNING: " + message);
-                // Checkstyle: resume
-            }
+            warningOrFail(message);
             Set<String> unknownAttributesForType = seenUnknownAttributesByType.computeIfAbsent(type, key -> new HashSet<>());
             unknownAttributesForType.addAll(unknownAttributes);
         }
     }
 
+    protected void warningOrFail(String message) {
```

Comment:
Maybe `warnOrFail` (to consistently use verbs)

Reply:
Fixed.